datafeeder/src/main/java/org/georchestra/datafeeder/autoconf/GeorchestraDatadirAutoConfiguration.java

```diff
@@ -20,19 +20,24 @@
 
 import javax.annotation.PostConstruct;
 
+import org.georchestra.datafeeder.app.YamlPropertySourceFactory;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.env.YamlPropertySourceLoader;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Profile;
 import org.springframework.context.annotation.PropertySource;
 
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.PropertySources;
 
 @Configuration
 @Profile("georchestra")
-@PropertySource(value = { //
-        "file:${georchestra.datadir}/default.properties", //
-        "file:${georchestra.datadir}/datafeeder/datafeeder.properties" }, //
-        ignoreResourceNotFound = false)
+@EnableConfigurationProperties(PostgisSchemasConfiguration.class)
+@PropertySources({ @PropertySource(value = { "file:${georchestra.datadir}/default.properties", //
+        "file:${georchestra.datadir}/datafeeder/datafeeder.properties" }, ignoreResourceNotFound = false),
+        @PropertySource(value = {
+                "file:${georchestra.datadir}/data-api/application.yaml" }, ignoreResourceNotFound = true, factory = YamlPropertySourceFactory.class) })
```

Comment:
I don't know if it would make sense to have it in its own dedicated class like `DataApiAutoConfiguration`, also with a system property dedicated to the data-api configuration path, instead of assuming it is into the georchestra datadir (there are no data-api configuration provided in the default georchestra datadir yet).

Reply:
Why not even a dedicated profile ? like "data-api" ?